gaphor/diagram/classes/tests/test_class.py

```diff
@@ -100,4 +100,4 @@ def test_compartment_resizing(self):
         diagram.canvas.update()
 
         width = klass.width
-        assert width > 170.0
+        assert width >= 170.0
```

Comment:
This looks okay to be. Testing the width doesn't make for a really stron test anyhow, since we depend on fonts and all.